src/features/click-to-load/components/ctl-placeholder-blocked.js

```diff
@@ -0,0 +1,281 @@
+     *      onClick: () => void // Share Feedback link click callback
+     *   }, // Shows feedback link on tablet and desktop sizes,
+     *   onButtonClick: (originalElement: HTMLIFrameElement | HTMLElement, replacementElement: HTMLElement) => (e: any) => void,
+     * }} params
+     */
+    constructor (params) {
+        super()
+        this.params = params
+        /**
+         * Create the shadow root, closed to prevent any outside observers
+         * @type {ShadowRoot}
+         */
+        const shadow = this.attachShadow({
+            mode: this.params.devMode ? 'open' : 'closed'
+        })
+
+        /**
+         * Add our styles
+         * @type {HTMLStyleElement}
+         */
+        const style = document.createElement('style')
+        style.innerText = css
+
+        /**
+         * Creates the placeholder for blocked content
+         * @type {HTMLDivElement}
+         */
+        this.placeholderBlocked = this.createPlaceholder()
+        /**
+         * Creates the Share Feedback element
+         * @type {HTMLDivElement | null}
+         */
+        const feedbackLink = this.params.withFeedback ? this.createShareFeedbackLink() : null
+        /**
+         * Setup the click handlers
+         */
+        this.setupEventListeners(this.placeholderBlocked, feedbackLink)
+
+        /**
+         * Append both to the shadow root
+         */
+        feedbackLink && this.placeholderBlocked.appendChild(feedbackLink)
+        shadow.appendChild(this.placeholderBlocked)
+        shadow.appendChild(style)
+    }
+
+    /**
+     * Creates a placeholder for content blocked by Click to Load.
+     * @returns {HTMLDivElement}
+     */
+    createPlaceholder = () => {
+        const { title, body, unblockBtnText, useSlimCard, withToggle, withFeedback } = this.params
+
+        const container = document.createElement('div')
+        container.classList.add('DuckDuckGoSocialContainer')
+
+        const learnMoreLink = this.createLearnMoreLink()
+
+        container.innerHTML = html`
+            <div
+                class="ddg-ctl-placeholder-card${useSlimCard ? ' slim-card' : ''}${withFeedback ? ' with-feedback-link' : ''}"
+            >
+                <div class="ddg-ctl-placeholder-card-header">
+                    <img class="ddg-ctl-placeholder-card-header-dax" src=${daxImg} alt="DuckDuckGo Dax" />
+                    <div class="ddg-ctl-placeholder-card-title">${title}. ${learnMoreLink}</div>
+                </div>
+                <div class="ddg-ctl-placeholder-card-body">
+                    <div class="ddg-ctl-placeholder-card-body-text">${body} ${learnMoreLink}</div>
+                    <button class="DuckDuckGoButton tertiary ddg-ctl-unblock-btn" type="button">
+                        <div>${unblockBtnText}</div>
+                    </button>
+                </div>
+                ${withToggle
+        ? html`<div class="ddg-ctl-placeholder-card-footer">${this.createToggleButton()}</div> `
+        : ''}
```

Comment:
The templating system here makes it difficult to do nice ternaries inline like this, I would extract this (and any others ) to in-scope variables instead, just to prevent having to deal with these formatting issues :)